node/test/Test/Pos/CborSpec.hs

```diff
@@ -223,11 +232,12 @@ roundtripProperty (input :: a) = ((deserialize . serialize $ input) :: a) === in
 -- without breaking anything. This should work with every time which adopted
 -- the schema of having at least one constructor of the form:
 -- .... | Unknown Word8 ByteString
-extensionProperty :: forall a. (Arbitrary a, Eq a, Show a, Bi a) => Property
-extensionProperty = forAll @a (arbitrary :: Gen a) $ \input ->
+extensionPropertyOn :: forall a b. (Eq a, Show a, Bi a, Arbitrary b, Eq b, Show b, Bi b)
+                    => Property
+extensionPropertyOn = forAll @b (arbitrary :: Gen b) $ \input ->
     let serialized      = serialize input -- We now have a BS blob
-        (u :: U)        = deserialize serialized
-        (encoded :: a)  = deserialize (serialize u)
+        (u :: a)        = deserialize serialized
+        (encoded :: b)  = deserialize (serialize u)
```

Comment:
Can we comment this property more clearly to explain how what we have here corresponds to the property we want to check. It's subtle and so not immediately obvious to the reader.